src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -7,6 +7,10 @@ import { kebabCase } from 'lodash';
 
 const analyticsEvents = {
   Modal: [{ action: 'show', event: 'int-modal-click' }],
+  AdditionalInfo: [
+    { action: 'expand', event: 'int-additional-info-expand' },
+    { action: 'contract', event: 'int-additional-info-contract' },
```

Comment:
`contract`, not `collapse`?

We don't need to worry about this at all? `src/applications/static-pages/subscribeAdditionalInfoEvents.js`

Reply:
> `contract`, not `collapse`?

Indeed. Fixed.

> We don't need to worry about this at all? `src/applications/static-pages/subscribeAdditionalInfoEvents.js`

Interesting!  I did not know there was analytics events in the formation package.  Correct me if I'm wrong, but this isn't using the actual component-library `AdditionalInfo` component, but more of a static site / Drupal recreation?

Reply:
@ncksllvn -

I took a look at one of the pages recording the formation events you linked (`/health-care/how-to-apply/index.html`) and it looks like it's separate.  I just see the formation events and not my new component library events...

http://35de3e34a2412cc9162c460c59689be8.review.vetsgov-internal/health-care/how-to-apply/index.html

```
monitorEvents(document.body, "component-library-analytics");
monitorEvents(document.body, '@department-of-veterans-affairs/formation/additional-info/expand');
monitorEvents(document.body, '@department-of-veterans-affairs/formation/additional-info/collapse');
```

![image](https://user-images.githubusercontent.com/62304138/112859871-42278800-9081-11eb-999e-3e233490081f.png)

Reply:
> Correct me if I'm wrong, but this isn't using the actual component-library AdditionalInfo component, but more of a static site / Drupal recreation?

you are correct!